examples/se2_inverse.py

```diff
@@ -0,0 +1,65 @@
+# This example will use torch to solve min_x1 || x1^-1 - x2 ||^2,
```

Comment:
Add license add the top and remember to update the readme in examples folder as necessary.

Reply:
Updated.